src/ts/lib/util/common.ts

```diff
@@ -14,6 +14,16 @@ class UtilCommon {
 		return window.Electron || {};
 	};
 
+	getCurrentElectronWindowId (): string {
+		const electron = this.getElectron();
+
+		if (!electron) {
+			return '0';
+		};
+
+		return electron.currentWindow().windowId.toString();
```

Comment:
Why is it a string?

Reply:
I thought that making it a string might act a bit as a safeguard against `windowId` maybe changing types in the future and causing problems when using it as a key in the local storage.